e2e/setup.go

```diff
@@ -113,6 +111,7 @@ type Cfg struct {
 	UseS3Caching       bool
 	UseRedisCaching    bool
 	UseS3Fallback      bool
+	WriteThreadCount   int
```

Comment:
prob makes the comment above ("at most one of the below options should be true") not true anymore. Maybe update the comment to say "below 3/4 options"

Reply:
[2d3559f](https://github.com/Layr-Labs/eigenda-proxy/pull/182/commits/2d3559f260ad130dbefd835f3ab13407a5494381)